packages/entities/entities-plugins/src/components/PluginEntityForm.vue

```diff
@@ -852,7 +861,7 @@ watch(() => props.schema, (newSchema, oldSchema) => {
   if (objectsAreEqual(newSchema || {}, oldSchema || {})) {
     return
   }
-  const form: Record<string, any> = parseSchema(newSchema)
+  const form: Record<string, any> = parseSchema(newSchema, undefined, undefined, props.engine as any)
```

Comment:
Not sure why we are casting it to any, maybe we should fix the typing in `parseSchema` instead?

Reply:
Because the type of `props.engine` is `string | undefined`